ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/approvedDeferralsTable.test.ts

```diff
@@ -0,0 +1,232 @@
+        cy.location('search').should(
+            'contain',
+            'sortOption[field]=Created%20Time&sortOption[direction]=desc'
+        );
+        cy.get(selectors.tableSortColumn('Requested')).should(
+            'have.attr',
+            'aria-sort',
+            'descending'
+        );
+        cy.get(selectors.tableColumnSortButton('Requested')).click();
+        cy.location('search').should(
+            'contain',
+            'sortOption[field]=Created%20Time&sortOption[direction]=asc'
+        );
+        cy.get(selectors.tableSortColumn('Requested')).should(
+            'have.attr',
+            'aria-sort',
+            'ascending'
+        );
+    });
+
+    it('should be able to sort on the "Expires" column', () => {
+        visitApprovedDeferralsTab();
+
+        cy.get(selectors.tableSortColumn('Expires')).should('have.attr', 'aria-sort', 'none');
+        cy.get(selectors.tableColumnSortButton('Expires')).click();
+        cy.location('search').should(
+            'contain',
+            'sortOption[field]=Request%20Expiry%20Time&sortOption[direction]=desc'
+        );
+        cy.get(selectors.tableSortColumn('Expires')).should('have.attr', 'aria-sort', 'descending');
+        cy.get(selectors.tableColumnSortButton('Expires')).click();
+        cy.location('search').should(
+            'contain',
+            'sortOption[field]=Request%20Expiry%20Time&sortOption[direction]=asc'
+        );
+        cy.get(selectors.tableSortColumn('Expires')).should('have.attr', 'aria-sort', 'ascending');
+    });
+
+    it('should be able to sort on the "Scope" column', () => {
+        visitApprovedDeferralsTab();
+
+        cy.get(selectors.tableSortColumn('Scope')).should('have.attr', 'aria-sort', 'none');
+        cy.get(selectors.tableColumnSortButton('Scope')).click();
+        cy.location('search').should(
+            'contain',
+            'sortOption[field]=Image%20Registry%20Scope&sortOption[direction]=desc'
+        );
+        cy.get(selectors.tableSortColumn('Scope')).should('have.attr', 'aria-sort', 'descending');
+        cy.get(selectors.tableColumnSortButton('Scope')).click();
+        cy.location('search').should(
+            'contain',
+            'sortOption[field]=Image%20Registry%20Scope&sortOption[direction]=asc'
+        );
+        cy.get(selectors.tableSortColumn('Scope')).should('have.attr', 'aria-sort', 'ascending');
+    });
+
+    it('should be able to filter by "Request name"', () => {
+        deferAndVisitRequestDetails({
+            comment,
+            expiry,
+            scope,
+        });
+        approveRequest();
+        visitApprovedDeferralsTab();
+
+        cy.get('table td[data-label="Request name"] a').then((element) => {
+            const requestName = element.text().trim();
+            typeAndSelectCustomSearchFilterValue('Request name', requestName);
+            cy.get('table td[data-label="Request name"] a').should('exist');
```

Comment:
What do you think about testing that a link that contains the actual request name here? I know we are cleaning up the tests in before/after hooks, but this could show a false-positive if there were other requests present in the system.

Additionally, it might be useful to test the inverse: filtering by a bogus request name results in an empty table.

Reply:
Could you explain that more? For this test, we don't require knowing the specific request. If there is a request A (could be any) in the table, and we filter by A, we should only see that request in the table. I think that would suffice. You're right about the inverse though. I think I did that for the "Requester". I should do it here too